var/spack/repos/builtin/packages/gslib/package.py

```diff
@@ -48,6 +52,12 @@ def install(self, spec, prefix):
 
         cc = self.compiler.cc
```

Comment:
@alalazo my IDE is saying "Cannot access attribute "cc" for class "DeprecatedCompiler"... I think this might reflect that no-one builds this library without MPI?

Reply:
Yeah I think it needs to change to `self["c"].cc`

Reply:
```suggestion
        cc = self["c"].cc
```

Like this? I think we can apply the suggestion and then merge...